test/statement_test.ts

```diff
@@ -30,9 +30,9 @@ describe('statements', () => {
   });
   it('translates try/catch', () => {
     expectTranslate('try {} catch(e) {} finally {}')
-        .to.equal(' try { } catch ( e ) { } finally { }');
+        .to.equal(' try { } catch ( e , e_stack ) { } finally { }');
```

Comment:
Or dart2js could detect the unused variable.

On Sun, May 10, 2015, 8:43 AM Martin Probst <notifications@github.com>
wrote:

> In test/statement_test.ts
> <https://github.com/angular/ts2dart/pull/153#discussion_r30003226>:
>
> > @@ -30,9 +30,9 @@ describe('statements', () => {
> >    });
> >    it('translates try/catch', () => {
> >      expectTranslate('try {} catch(e) {} finally {}')
> > -        .to.equal(' try { } catch ( e ) { } finally { }');
> > +        .to.equal(' try { } catch ( e , e_stack ) { } finally { }');
>
> @mhevery <https://github.com/mhevery> if catching the stack is that
> expensive, surely the VM optimizes for the case where the stack isn't
> actually used subsequently, and we can just generate stupid code that
> always captures the stack?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/angular/ts2dart/pull/153/files#r30003226>.
>